collection-scripts/gather-istio

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Resource List
+resources=()
+
+# Istio Resources
+resources+=(ns/istio-operator ns/istio-system)
+
+# Custom Resources
+resources+=(CustomResourceDefinition)
```

Comment:
We likely don't want to collect this. We likely want to collect the specific ISTIO CRD and or Active CR. 

We probably want to have this pull `clusteroperator/ISTIO` or something like that.